app/communication/views/banner.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework import viewsets
+
+from app.common.pagination import BasePagination
+from app.common.permissions import BasicViewPermission
+from app.communication.models.banner import Banner
+from app.communication.serializers.banner import BannerSerializer
+
+
+class BannerViewSet(viewsets.ModelViewSet):
```

Comment:
Her bør vi ha en måte for admins å kunne hente ut alle bannere, samt at "vanlige" brukere kan hente ut "dagens banner" om det finnes